packages/apps/custom-payment/src/custom-payment-create-transaction.ts

```diff
@@ -0,0 +1,65 @@
+import type { AppModuleBody } from '@cloudcommerce/types';
+import type { CreateTransactionParams } from '@cloudcommerce/types/modules/create_transaction:params';
+import type { CreateTransactionResponse } from '@cloudcommerce/types/modules/create_transaction:response';
```

Comment:
```suggestion
import type {
  AppModuleBody,
  CreateTransactionParams,
  CreateTransactionResponse,
} from '@cloudcommerce/types';
```

Reply:
Não tá errado, mas não mais fácil assim não? 😛

Reply:
https://github.com/ecomplus/cloud-commerce/blob/main/packages/types/index.ts#L138-L139

Reply:
Nooo comi mosca demais aqui. :cry: 
Logo em seguida vou abrir uma PR para fixar isso nos apps que portei e ficaram assim.

Reply:
Pdc, mas quando for mexer nisso quebra as linhas também:

```js
import type {
  AppModuleBody,
  CreateTransactionParams,
  CreateTransactionResponse,
} from '@cloudcommerce/types';
```

Pra não exceder o máximo de 100 do https://github.com/airbnb/javascript